docs/invoking-tester/cli-reference.rst

```diff
@@ -8,15 +8,46 @@ Basic usage
 
    .. code-block:: bash
                    
-      sampletester TEST.yaml [TEST.yaml ...] [MANIFEST.manifest.yaml ...]
+      sampletester CONFIG_PATH [CONFIG_PATH ...]
```

Comment:
The issue linked only requires sampletester to take at most two parameters, with the first one being test config pattern and the second one being the manifest pattern, and both of which are optional.

The description here allows an arbitrary number of inputs, and test configs don't necessarily have to precede the manifests, thus the requirements of parameters are looser than that described in the issue. Is this OK?

Reply:
#52 implies that there can be more than two parameters. The way I read it, any number of directories or files can be specified. This is the case currently (before this PR), as is the lack of relative order between manifests and testplans (prior to this PR, they were disambiguated solely on the basis of filename).

Additions on top of #52 are that these parameters are treated as globs, so the caller need not rely on shell globbing, and that the yaml documents are self-describing by default. I'll add this to the PR description.

Reply:
Ok that makes sense.